src/site/lab9.rst

```diff
@@ -0,0 +1,137 @@
+******
+Lab #9
+******
+
+* Refer to the topic pages for this lab
+    * You will find many of the answers to your questions there
+* Feel free to use your laptop if you have it
+* I strongly encourage you to work with others in the lab
+* Ensure I have recorded your completion. Failure to do so will result in a grade of 0
+* When you get stuck, do me a favour and ask those sitting around you for help
+    * I want people to get used to working together in the labs
+    * Peer teaching and peer learning is super effective
+
+
+Draw The Tree
+=============
+
+
+Binary Node
+===========
+
+
+
+.. code-block:: java
+
+    public static void main(String[] args){
+        BinaryNode<Integer> root = new BinaryNode<>(1);
```

Comment:
Is the intention here that the tree be actually in order for the in order traversal? left-visit-right gives a jumbling ordering, whereas pre-order will actually be in order here.

Reply:
It was not intended, but it's a great idea --- will change.